### PR TITLE
Add git revision footer to every non-preso page

### PR DESCRIPTION
**Pull request by liam-middlebrook**
I'm not sure this visually looks the best that it could. But I think that this is an important thing to have in order to know what version of conditional is actively being run in a wsgi process.

### CODE REVIEW

**Comment by stevenmirabito** [comment]
Lgtm. Is the whitespace change in `flask.py` a PEP8 fix or accidental?

**Comment by liam-middlebrook** [comment]
Nope, just fixed that.

**Comment by mbillow** [comment]
I made this look not ugly. 
screen shot 2016-12-28 at 23 01 24

I am ready to merge this in, if you guys are.

File: conditional/templates/base.html [hunk 1]
```diff
@@ -28,8 +28,9 @@
         </div>
         {% endif %}
 
+        <footer>
+            <a class="footer-version" href="https://github.com/ComputerScienceHouse/conditional">Conditional ({{config["GIT_REVISION"]}})</a>
```
Comment (liam-middlebrook) [inline conditional/templates/base.html]: Can we make the href also include the commit hash? So it links directly to browse at that part?

Comment (liam-middlebrook) [inline conditional/templates/base.html]: The format would be `https://github.com/ComputerScienceHouse/conditional/tree/$commitHash`

**Comment by stevenmirabito** [comment]
Looks good pending the change requested by @liam-middlebrook

**Pull request merged**